mscore/metaedit.cpp

```diff
@@ -182,15 +182,15 @@ void MetaEditDialog::setDirty(const bool dirty)
       }
 
 //---------------------------------------------------------
-//   setDirty
+//   openFileLocation
 ///    Opens the file location with a QMessageBox::warning on failure
 //---------------------------------------------------------
 
 void MetaEditDialog::openFileLocation()
       {
       if (!OpenFileLocation::openFileLocation(filePath->text()))
-          QMessageBox::warning(this, QString(QT_TR_NOOP("Open Containing Folder Error")),
-                                     QString(QT_TR_NOOP("Could not open containing folder")));
+            QMessageBox::warning(this, QString(tr("Open Containing Folder Error")),
+                                       QString(tr("Could not open containing folder")));
```

Comment:
These `QString()` conversions are not needed here anymore, `tr()` function already returns a `QString`.

Reply:
True, fixed